docs/concepts/specifications/reserved-namespace-id.md

```diff
@@ -7,13 +7,19 @@ description: Reserved namespace IDs on the Celestia Network.
 
 This is a table of reserved namespace IDs on the Celestia Network.
```

Comment:
[not blocking] proposal to drop the "ID" portion from the top of this page because after celestia-app [ADR 014](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-014-versioned-namespaces.md) namespace is now composed of a version and an ID.

Sorry I can't include a Github suggestion b/c these lines haven't been modified but basically:

```markdown
---
sidebar_label: Reserved Namespaces
description: Reserved Namespaces on the Celestia Network.
---
# Reserved Namespaces

This is a table of reserved namespaces on the Celestia Network.
```

Reply:
yes, thanks for catching this, too